lib/utils.ts

```diff
@@ -20,7 +20,7 @@ export function camelCaseID(id: string) {
 }
 
 interface MinMaxInterface {
-  type: 'min' | 'max';
+  type: 'min' | 'max' | 'constant';
   index: number;
 }
```

Comment:
This change is great, but it also changes the responsibility of this function quite a bit. Seems like now, we're basically returning the graph's trend by adding the possibility of the graph being `constant`, and deviating from the behavior of just returning the min or max. Accordingly, I think we should refactor this function (and this interface) to just return the trend of the graph, whether it's increasing, decreasing, or constant. 

```TypeScript
interface GraphTrendInterface {
   behavior: 'increasing' | 'decreasing' | 'constant';
   endIndex: number;
}
```